queries.sql

```diff
@@ -43,3 +43,96 @@ FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. 
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+SELECT * FROM animals; -- species column = unspecified
+ROLLBACK;
+
+--Inside a transaction: Update the animals table by setting the species column to digimon for all animals that have a name ending in mon. Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+
+BEGIN;
+
+-- Update species to 'digimon' for animals with names ending in "mon"
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update species to 'pokemon' for animals with no species set
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
```

Comment:
👏 Great job on working through the transaction steps! However, as the previous reviewer mentioned, to align with the project requirements, it's important to verify that the changes persist after the commit. As per the transaction's components, the last part is:

- [ ] Verify that changes persist after commit. 🔴

In your transaction, you've successfully completed the first four parts, but you need to verify that the changes made in the transaction persist after the commit. To do this, you should perform a select query to check the data in the table and ensure that it matches your expectations.

Here's an example of how to do it:

```sql
-- Verify that changes persist after commit
SELECT * FROM animals;
```

By adding this step to your transaction and checking that the data remains as expected after the commit, you'll fulfill the project requirement. 

- [ ] Also, please update your screenshot as well as the transection in the `queries.sql` file to include this final part of the transaction. This will provide a comprehensive overview of the entire transaction process. Keep up the good work! 👍📊🔍